stubs/django/test/__init__.pyi

```diff
@@ -11,6 +11,5 @@ from django.test.client import Client as Client
 from django.test.utils import override_settings as override_settings
 
 class SimpleTestCase(TestCase):
-    client: testing.client.base.TestClient = ...
-    @contextlib.contextmanager
```

Comment:
Did you intend to delete this line? Looks like they do say `settings` is a context manager:
https://github.com/django/django/blob/dbb4a86fa7c15fb9cf73a5954356eb88d65ac25f/django/test/testcases.py#L307

Reply:
My attention was drawn here by the missing import. `settings` itself isn't defined as a context manager; its return value is. That's what tripped me up. I'll add the import and put the decorator back.